9.1.0.246/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.10.2
+
+ENV NR_PORT /tmp/.newrelic.sock
+
+ARG NR_VERSION=9.1.0.246
+ENV NR_FILE_NAME=newrelic-php5-$NR_VERSION-linux-musl
+ENV NR_FILE_URL=https://download.newrelic.com/php_agent/archive/$NR_VERSION/$NR_FILE_NAME.tar.gz
+
+RUN set -ex \
+    && apk update \
```

Comment:
really needed ?

Reply:
Yes, it was needed to prevent APKINDEX errors during the build of the image.

This is described here https://github.com/gliderlabs/docker-alpine/issues/207. In this case this is described as a simple "WARNING" but it is not the case and prevent the build to succeed.